docs/Useful-scripts.rst

```diff
@@ -81,3 +81,24 @@ This results in ``broken.eb`` being rewritten/fixed, after creating a backup cop
 
 A script to set up an EasyBuild development environment.
 For more information, see :ref:`install_latest_develop_using_script`.
+
+.. _clean_gists_script:
+
+``clean_gists.py``
+--------------------------------
+
+*download from:* https://github.com/hpcugent/easybuild-framework/blob/master/easybuild/scripts/clean_gists.py
+
+When using ``--from-pr`` in combination with ``-upload-test-report`` (see
+https://github.com/hpcugent/easybuild/wiki/Review-process-for-contributions#automated-testing-of-easyconfigs-pull-requests),
+you can end up with a bunch of gists in your GitHub account containing test reports, that may no longer be relevant.
+
+To help with that the ``clean_gists.py`` script is available, to clean up gists containing test reports:
+
+* ``clean_gists.py -p``: delete all gists from closed pull requests (default action if no other action is specified)
+* ``clean_gists.py -a``: delete all gists generated by Easybuild
+* ``clean_gists.py -o``: delete all gists without a matching pull request (created by using ``-upload-test-report`` without ``--from-pr``)
+
+By default, the script will use the same GitHub account that Easybuild uses (see ``--github-user``); to specify a different GitHub account, use ``-g``.
+
+The script expects that a valid GitHub token for the used GitHub account username is available, see ref:`github_token`.
```

Comment:
hmm, weird... seems like cross-references labels across pages isn't working like I thought it was